src/tabs/uploadCV.svelte

```diff
@@ -1,51 +1,52 @@
 <script>
-  import { getDocument, GlobalWorkerOptions } from "pdfjs-dist";
-  import { userCV } from "~lib/stores/userCV";
+  import { getDocument, GlobalWorkerOptions } from "pdfjs-dist"
+
+  import { userCV } from "~lib/stores/userCV"
 
   async function handleCVUpload(event) {
-    const file = event.target.files[0];
-    const reader = new FileReader();
+    const file = event.target.files[0]
+    const reader = new FileReader()
     GlobalWorkerOptions.workerSrc = await import(
       "pdfjs-dist/build/pdf.worker.entry"
-    );
+    )
 
-    reader.onload = (e) => ParsePDF(e, userCV.set);
-    reader.readAsArrayBuffer(file);
+    reader.onload = (e) => ParsePDF(e, userCV.set)
+    reader.readAsArrayBuffer(file)
   }
 
   async function handleDeleteCV() {
-    await userCV.set("");
+    await userCV.set("")
   }
 
   async function ParsePDF(e, setData) {
-    const pdfData = e.target.result;
-    const pdf = await getDocument(pdfData).promise;
-    const maxPages = 4;
-    let extractedText = "";
+    const pdfData = e.target.result
+    const pdf = await getDocument(pdfData).promise
+    const maxPages = Math.min(4, pdf.numPages)
```

Comment:
I want to keep the maximum pages at 4, to keep API costs low. If the user uploads a 150 page PDF book, I don't want to pay for the tokens to openAI for that.

Can you add an `window.alert` or similar to notify the user, that the PDF is too big instead?